### ISSUE TITLE
apache getting stop if we try to connect mongodb?

### ISSUE THREAD

**Issue by gaikwadrg95** [mongodb/mongo-php-driver#492]
### Description

I am new with mongodb, I am trying to connect mongodb with php using reference [youtub video](https://www.youtube.com/watch?v=6mCgckvPScI)

When  I run code the green signal of apache on XAMPP control panel turns to yellow for few seconds and on a browser it will display the message
"This site can’t be reached"

What can be done?
Thanks.
### Environment

operating system=windows 8.1 and 
PHP version=7.0.9 
PHP was installed through a third-party bundle XAMPP.

### Test Script
```
<?php
require 'vendor/autoload.php';

$myclient=new MongoDB\Client;
echo "successfull...";
?>
```

**Comment by lindelius** [comment]
Which driver version are you using?

**Comment by gaikwadrg95** [comment]
thank lindelius,
i upgraded DLL to 1.2.2,but problem is not solved, now its showing following error

" Fatal error: Uncaught Error: Class 'MongoDB\Driver\Manager' not found in E:\Project_NS\Software\xamm\htdocs\phpmongodb\vendor\mongodb\mongodb\src\Client.php:66"

**Comment by lindelius** [comment]
Which version of the DLL did you include? It should have been the PHP 7.0 TS x86 version.

**Comment by gaikwadrg95** [comment]
I included PHP 7.0 TS x64 version because my OS is 64-bit.
